src/ServicePulse.Host/app/js/views/archive/view.html

```diff
@@ -51,7 +51,7 @@
                     </div>
 
                     <div class="col-xs-11 failed-message-data" ng-mouseenter="message.hover2 = true" ng-mouseleave="message.hover2 = false">
-                        <a href="#/failedMessages/{{message.id}}">
+                        <a href="#/message/{{message.id}}">
```

Comment:
It is and it isn't :)
Previously our URL's looked as follows:
/failedMessages - failed mesasges screen
/failedMessages/{{id}} - one message screen

Becasue @WilliamBZA made a pr to provide deep linking groups he used "our" url path, so currently it looks like this:
/failedMessages - failed mesasges screen
/failedMessages/{{id}} - failed messages screen with a group of given id selected

Which means that we lacked url for this feature. Now becasue on our page we show a bit more than only failed message (as it can also be archived message, or succesfful processed message) I decided to go with path:
/message/{{id}}

What do you think?